### ISSUE TITLE
distcc: service runs wrong command on server

### ISSUE THREAD

**Issue by z-aki** [Homebrew/homebrew-core#129930]
### `brew gist-logs <formula>` link OR `brew config` AND `brew doctor` output

<details>
<summary> logs </summary>

```
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry or file an issue; just ignore this. Thanks!

Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  php@7.4

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/node/cppgc/allocation.h
  /usr/local/include/node/cppgc/common.h
  /usr/local/include/node/cppgc/custom-space.h
  /usr/local/include/node/cppgc/garbage-collected.h
  /usr/local/include/node/cppgc/heap.h
  /usr/local/include/node/cppgc/internal/accessors.h
  /usr/local/include/node/cppgc/internal/api-constants.h
  /usr/local/include/node/cppgc/internal/compiler-specific.h
  /usr/local/include/node/cppgc/internal/finalizer-trait.h
  /usr/local/include/node/cppgc/internal/gc-info.h
  /usr/local/include/node/cppgc/internal/logging.h
  /usr/local/include/node/cppgc/internal/persistent-node.h
  /usr/local/include/node/cppgc/internal/pointer-policies.h
  /usr/local/include/node/cppgc/internal/prefinalizer-handler.h
  /usr/local/include/node/cppgc/liveness-broker.h
  /usr/local/include/node/cppgc/macros.h
  /usr/local/include/node/cppgc/member.h
  /usr/local/include/node/cppgc/persistent.h
  /usr/local/include/node/cppgc/platform.h
  /usr/local/include/node/cppgc/prefinalizer.h
  /usr/local/include/node/cppgc/source-location.h
  /usr/local/include/node/cppgc/trace-trait.h
  /usr/local/include/node/cppgc/type-traits.h
  /usr/local/include/node/cppgc/visitor.h
  /usr/local/include/node/js_native_api.h
  /usr/local/include/node/js_native_api_types.h
  /usr/local/include/node/libplatform/libplatform-export.h
  /usr/local/include/node/libplatform/libplatform.h
  /usr/local/include/node/libplatform/v8-tracing.h
  /usr/local/include/node/node.h
  /usr/local/include/node/node_api.h
  /usr/local/include/node/node_api_types.h
  /usr/local/include/node/node_buffer.h
  /usr/local/include/node/node_object_wrap.h
  /usr/local/include/node/node_version.h
  /usr/local/include/node/openssl/aes.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm/include/progs.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/BSD-x86/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/BSD-x86/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/BSD-x86/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/BSD-x86/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm/include/progs.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/BSD-x86_64/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm/include/progs.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/VC-WIN32/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/VC-WIN32/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/VC-WIN64-ARM/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/VC-WIN64-ARM/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64-ARM/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64-ARM/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/VC-WIN64-ARM/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm/include/progs.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/VC-WIN64A/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/aix-gcc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/aix-gcc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/aix-gcc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/aix-gcc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/aix-gcc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/aix-gcc/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/aix64-gcc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/aix64-gcc/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin-i386-cc/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin64-arm64-cc/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/darwin64-x86_64-cc/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-aarch64/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-aarch64/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-armv4/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-armv4/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-armv4/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-armv4/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-armv4/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-armv4/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-elf/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-elf/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-elf/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-elf/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-elf/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-elf/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-elf/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-elf/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc64/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc64/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-ppc64le/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-x32/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-x32/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-x32/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-x32/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-x32/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-x32/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-x32/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-x32/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux-x86_64/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux-x86_64/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux32-s390x/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux32-s390x/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux64-mips64/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux64-mips64/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm/include/progs.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/linux64-s390x/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/linux64-s390x/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/solaris-x86-gcc/no-asm/include/progs.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm/include/progs.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm_avx2/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm_avx2/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm_avx2/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm_avx2/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/asm_avx2/include/progs.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/no-asm/crypto/buildinf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/no-asm/crypto/include/internal/bn_conf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/no-asm/crypto/include/internal/dso_conf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/no-asm/include/openssl/opensslconf.h
  /usr/local/include/node/openssl/archs/solaris64-x86_64-gcc/no-asm/include/progs.h
  /usr/local/include/node/openssl/asn1.h
  /usr/local/include/node/openssl/asn1_mac.h
  /usr/local/include/node/openssl/asn1err.h
  /usr/local/include/node/openssl/asn1t.h
  /usr/local/include/node/openssl/async.h
  /usr/local/include/node/openssl/asyncerr.h
  /usr/local/include/node/openssl/bio.h
  /usr/local/include/node/openssl/bioerr.h
  /usr/local/include/node/openssl/blowfish.h
  /usr/local/include/node/openssl/bn.h
  /usr/local/include/node/openssl/bn_conf.h
  /usr/local/include/node/openssl/bn_conf_asm.h
  /usr/local/include/node/openssl/bn_conf_no-asm.h
  /usr/local/include/node/openssl/bnerr.h
  /usr/local/include/node/openssl/buffer.h
  /usr/local/include/node/openssl/buffererr.h
  /usr/local/include/node/openssl/camellia.h
  /usr/local/include/node/openssl/cast.h
  /usr/local/include/node/openssl/cmac.h
  /usr/local/include/node/openssl/cms.h
  /usr/local/include/node/openssl/cmserr.h
  /usr/local/include/node/openssl/comp.h
  /usr/local/include/node/openssl/comperr.h
  /usr/local/include/node/openssl/conf.h
  /usr/local/include/node/openssl/conf_api.h
  /usr/local/include/node/openssl/conferr.h
  /usr/local/include/node/openssl/crypto.h
  /usr/local/include/node/openssl/cryptoerr.h
  /usr/local/include/node/openssl/ct.h
  /usr/local/include/node/openssl/cterr.h
  /usr/local/include/node/openssl/des.h
  /usr/local/include/node/openssl/dh.h
  /usr/local/include/node/openssl/dherr.h
  /usr/local/include/node/openssl/dsa.h
  /usr/local/include/node/openssl/dsaerr.h
  /usr/local/include/node/openssl/dso_conf.h
  /usr/local/include/node/openssl/dso_conf_asm.h
  /usr/local/include/node/openssl/dso_conf_no-asm.h
  /usr/local/include/node/openssl/dtls1.h
  /usr/local/include/node/openssl/e_os2.h
  /usr/local/include/node/openssl/ebcdic.h
  /usr/local/include/node/openssl/ec.h
  /usr/local/include/node/openssl/ecdh.h
  /usr/local/include/node/openssl/ecdsa.h
  /usr/local/include/node/openssl/ecerr.h
  /usr/local/include/node/openssl/engine.h
  /usr/local/include/node/openssl/engineerr.h
  /usr/local/include/node/openssl/err.h
  /usr/local/include/node/openssl/evp.h
  /usr/local/include/node/openssl/evperr.h
  /usr/local/include/node/openssl/hmac.h
  /usr/local/include/node/openssl/idea.h
  /usr/local/include/node/openssl/kdf.h
  /usr/local/include/node/openssl/kdferr.h
  /usr/local/include/node/openssl/lhash.h
  /usr/local/include/node/openssl/md2.h
  /usr/local/include/node/openssl/md4.h
  /usr/local/include/node/openssl/md5.h
  /usr/local/include/node/openssl/mdc2.h
  /usr/local/include/node/openssl/modes.h
  /usr/local/include/node/openssl/obj_mac.h
  /usr/local/include/node/openssl/objects.h
  /usr/local/include/node/openssl/objectserr.h
  /usr/local/include/node/openssl/ocsp.h
  /usr/local/include/node/openssl/ocsperr.h
  /usr/local/include/node/openssl/opensslconf.h
  /usr/local/include/node/openssl/opensslconf_asm.h
  /usr/local/include/node/openssl/opensslconf_no-asm.h
  /usr/local/include/node/openssl/opensslv.h
  /usr/local/include/node/openssl/ossl_typ.h
  /usr/local/include/node/openssl/pem.h
  /usr/local/include/node/openssl/pem2.h
  /usr/local/include/node/openssl/pemerr.h
  /usr/local/include/node/openssl/pkcs12.h
  /usr/local/include/node/openssl/pkcs12err.h
  /usr/local/include/node/openssl/pkcs7.h
  /usr/local/include/node/openssl/pkcs7err.h
  /usr/local/include/node/openssl/rand.h
  /usr/local/include/node/openssl/rand_drbg.h
  /usr/local/include/node/openssl/randerr.h
  /usr/local/include/node/openssl/rc2.h
  /usr/local/include/node/openssl/rc4.h
  /usr/local/include/node/openssl/rc5.h
  /usr/local/include/node/openssl/ripemd.h
  /usr/local/include/node/openssl/rsa.h
  /usr/local/include/node/openssl/rsaerr.h
  /usr/local/include/node/openssl/safestack.h
  /usr/local/include/node/openssl/seed.h
  /usr/local/include/node/openssl/sha.h
  /usr/local/include/node/openssl/srp.h
  /usr/local/include/node/openssl/srtp.h
  /usr/local/include/node/openssl/ssl.h
  /usr/local/include/node/openssl/ssl2.h
  /usr/local/include/node/openssl/ssl3.h
  /usr/local/include/node/openssl/sslerr.h
  /usr/local/include/node/openssl/stack.h
  /usr/local/include/node/openssl/store.h
  /usr/local/include/node/openssl/storeerr.h
  /usr/local/include/node/openssl/symhacks.h
  /usr/local/include/node/openssl/tls1.h
  /usr/local/include/node/openssl/ts.h
  /usr/local/include/node/openssl/tserr.h
  /usr/local/include/node/openssl/txt_db.h
  /usr/local/include/node/openssl/ui.h
  /usr/local/include/node/openssl/uierr.h
  /usr/local/include/node/openssl/whrlpool.h
  /usr/local/include/node/openssl/x509.h
  /usr/local/include/node/openssl/x509_vfy.h
  /usr/local/include/node/openssl/x509err.h
  /usr/local/include/node/openssl/x509v3.h
  /usr/local/include/node/openssl/x509v3err.h
  /usr/local/include/node/uv.h
  /usr/local/include/node/uv/aix.h
  /usr/local/include/node/uv/android-ifaddrs.h
  /usr/local/include/node/uv/bsd.h
  /usr/local/include/node/uv/darwin.h
  /usr/local/include/node/uv/errno.h
  /usr/local/include/node/uv/linux.h
  /usr/local/include/node/uv/os390.h
  /usr/local/include/node/uv/posix.h
  /usr/local/include/node/uv/stdint-msvc2008.h
  /usr/local/include/node/uv/sunos.h
  /usr/local/include/node/uv/threadpool.h
  /usr/local/include/node/uv/tree.h
  /usr/local/include/node/uv/unix.h
  /usr/local/include/node/uv/version.h
  /usr/local/include/node/uv/win.h
  /usr/local/include/node/v8-fast-api-calls.h
  /usr/local/include/node/v8-internal.h
  /usr/local/include/node/v8-platform.h
  /usr/local/include/node/v8-profiler.h
  /usr/local/include/node/v8-util.h
  /usr/local/include/node/v8-value-serializer-version.h
  /usr/local/include/node/v8-version-string.h
  /usr/local/include/node/v8-version.h
  /usr/local/include/node/v8-wasm-trap-handler-posix.h
  /usr/local/include/node/v8-wasm-trap-handler-win.h
  /usr/local/include/node/v8.h
  /usr/local/include/node/v8config.h
  /usr/local/include/node/zconf.h
  /usr/local/include/node/zlib.h

Warning: Unbrewed '.la' files were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected '.la' files:
  /usr/local/lib/libprofiler.la
  /usr/local/lib/libtcmalloc.la
  /usr/local/lib/libtcmalloc_and_profiler.la
  /usr/local/lib/libtcmalloc_debug.la
  /usr/local/lib/libtcmalloc_minimal.la
  /usr/local/lib/libtcmalloc_minimal_debug.la

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libabsl_bad_any_cast_impl.a
  /usr/local/lib/libabsl_bad_optional_access.a
  /usr/local/lib/libabsl_bad_variant_access.a
  /usr/local/lib/libabsl_base.a
  /usr/local/lib/libabsl_city.a
  /usr/local/lib/libabsl_civil_time.a
  /usr/local/lib/libabsl_cord.a
  /usr/local/lib/libabsl_debugging_internal.a
  /usr/local/lib/libabsl_demangle_internal.a
  /usr/local/lib/libabsl_dynamic_annotations.a
  /usr/local/lib/libabsl_examine_stack.a
  /usr/local/lib/libabsl_exponential_biased.a
  /usr/local/lib/libabsl_failure_signal_handler.a
  /usr/local/lib/libabsl_flags.a
  /usr/local/lib/libabsl_flags_config.a
  /usr/local/lib/libabsl_flags_internal.a
  /usr/local/lib/libabsl_flags_marshalling.a
  /usr/local/lib/libabsl_flags_parse.a
  /usr/local/lib/libabsl_flags_program_name.a
  /usr/local/lib/libabsl_flags_registry.a
  /usr/local/lib/libabsl_flags_usage.a
  /usr/local/lib/libabsl_flags_usage_internal.a
  /usr/local/lib/libabsl_graphcycles_internal.a
  /usr/local/lib/libabsl_hash.a
  /usr/local/lib/libabsl_hashtablez_sampler.a
  /usr/local/lib/libabsl_int128.a
  /usr/local/lib/libabsl_leak_check.a
  /usr/local/lib/libabsl_leak_check_disable.a
  /usr/local/lib/libabsl_log_severity.a
  /usr/local/lib/libabsl_malloc_internal.a
  /usr/local/lib/libabsl_periodic_sampler.a
  /usr/local/lib/libabsl_random_distributions.a
  /usr/local/lib/libabsl_random_internal_distribution_test_util.a
  /usr/local/lib/libabsl_random_internal_pool_urbg.a
  /usr/local/lib/libabsl_random_internal_randen.a
  /usr/local/lib/libabsl_random_internal_randen_hwaes.a
  /usr/local/lib/libabsl_random_internal_randen_hwaes_impl.a
  /usr/local/lib/libabsl_random_internal_randen_slow.a
  /usr/local/lib/libabsl_random_internal_seed_material.a
  /usr/local/lib/libabsl_random_seed_gen_exception.a
  /usr/local/lib/libabsl_random_seed_sequences.a
  /usr/local/lib/libabsl_raw_hash_set.a
  /usr/local/lib/libabsl_raw_logging_internal.a
  /usr/local/lib/libabsl_scoped_set_env.a
  /usr/local/lib/libabsl_spinlock_wait.a
  /usr/local/lib/libabsl_stacktrace.a
  /usr/local/lib/libabsl_status.a
  /usr/local/lib/libabsl_str_format_internal.a
  /usr/local/lib/libabsl_strings.a
  /usr/local/lib/libabsl_strings_internal.a
  /usr/local/lib/libabsl_symbolize.a
  /usr/local/lib/libabsl_synchronization.a
  /usr/local/lib/libabsl_throw_delegate.a
  /usr/local/lib/libabsl_time.a
  /usr/local/lib/libabsl_time_zone.a

Warning: Homebrew's "sbin" was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting your PATH for example like so:
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.zshrc

Warning: Broken symlinks were found. Remove them with `brew cleanup`:
  /usr/local/bin/hub-tool
  /usr/local/bin/kubectl
  /usr/local/bin/kubectl.docker
  /usr/local/bin/vpnkit

brew config
HOMEBREW_VERSION: 4.0.16-2-gcdc5463
ORIGIN: https://github.com/Homebrew/brew
HEAD: cdc54636ee201cfaf7fe1a0148aaf8c29ad65dad
Last commit: 16 hours ago
Core tap origin: https://github.com/Homebrew/homebrew-core
Core tap HEAD: ed6d776eabc40a5198364e87f634965e9cbe067a
Core tap last commit: 9 weeks ago
Core tap branch: master
Core tap JSON: 02 May 06:04 UTC
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_DISPLAY: /private/tmp/com.apple.launchd.nnP5rDvwfy/org.xquartz:0
HOMEBREW_EDITOR: nano
HOMEBREW_MAKE_JOBS: 4
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: quad-core 64-bit broadwell
Clang: 14.0.0 build 1400
Git: 2.37.1 => /Library/Developer/CommandLineTools/usr/bin/git
Curl: 7.87.0 => /usr/bin/curl
macOS: 12.6.4-x86_64
CLT: 14.2.0.0.1.1668646533
Xcode: N/A

```
</details>

### Verification

- [ ] My "`brew doctor` output" says `Your system is ready to brew.` and am still able to reproduce my issue.
- [X] I ran `brew update` and am still able to reproduce my issue.
- [X] I have resolved all warnings from `brew doctor` and that did not fix my problem.
- [X] I searched for recent similar issues at https://github.com/Homebrew/homebrew-core/issues?q=is%3Aissue and found no duplicates.

### What were you trying to do (and why)?

I was trying to run distcc on a build server

### What happened (include all command output)?

distcc error 256

because it ran run [opt_bin/"distcc", "--allow=192.168.0.1/24"]

### What did you expect to happen?

Server should've run 

run [opt_bin/"distccd", "--allow=192.168.0.1/24"]

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
brew install distcc
brew services start distcc
```

**Comment by SMillerDev** [comment]
How would it know that it's a server?

**Comment by z-aki** [comment]
how would [opt_bin/"distcc", "--allow=192.168.0.1/24"] be useful for a client ?

**Comment by z-aki** [comment]
```
distcc --help                                                                                                                                                               
distcc 3.4 x86_64-apple-darwin21.6.0
  (protocols 1, 2 and 3) (default port 3632)
  built Jan 15 2023 17:28:42
Copyright (C) 2002, 2003, 2004 by Martin Pool.
Includes miniLZO (C) 1996-2002 by Markus Franz Xaver Johannes Oberhumer.
Portions Copyright (C) 2007-2008 Google.

distcc comes with ABSOLUTELY NO WARRANTY.  distcc is free software, and
you may use, modify and redistribute it under the terms of the GNU 
General Public License version 2 or later.

Please report bugs to distcc@lists.samba.org

Usage:
   distcc [--scan-includes] [COMPILER] [compile options] -o OBJECT -c SOURCE
   distcc [--help|--version|--show-hosts|-j]

Options:
   COMPILER                   Defaults to "cc".
   --help                     Explain usage, and exit.
   --version                  Show version, and exit.
   --show-hosts               Show host list, and exit.
   -j                         Show the concurrency level, as calculated from
                              the host list, and exit.
   --scan-includes            Show the files that distcc would send to the
                              remote machine, and exit.  (Pump mode only.)

Environment variables:
   See the manual page for a complete list.
   DISTCC_VERBOSE=1           Give debug messages.
   DISTCC_LOG                 Send messages to file, not stderr.
   DISTCC_SSH                 Command to run to open SSH connections.
   DISTCC_DIR                 Directory for host list and locks.

Server specification:
A list of servers is taken from the environment variable $DISTCC_HOSTS, or
$DISTCC_DIR/hosts, or ~/.distcc/hosts, or /usr/local/Cellar/distcc/3.4_1/etc/distcc/hosts.
Each host can be given in any of these forms, see the manual for details:

   localhost                  Run in place.
   HOST                       TCP connection, port 3632.
   HOST:PORT                  TCP connection, specified port.
   @HOST                      SSH connection to specified host.
   USER@HOST                  SSH connection to specified username at host.
   HOSTSPEC,lzo               Enable compression.
   HOSTSPEC,cpp,lzo           Use pump mode (remote preprocessing).
   HOSTSPEC,auth              Enable GSS-API based mutual authenticaton.
   --randomize                Randomize the server list before execution.

distcc distributes compilation jobs across volunteer machines running
distccd.  Jobs that cannot be distributed, such as linking, are run locally.
distcc should be used with make's -jN option to execute in parallel on
several machines.

$ distccd --help                                                                                                                                                              
distccd 3.4 x86_64-apple-darwin21.6.0
  (protocols 1, 2 and 3) (default port 3632)
  built Jan 15 2023 17:28:42
Copyright (C) 2002, 2003, 2004 by Martin Pool.
Includes miniLZO (C) 1996-2002 by Markus Franz Xaver Johannes Oberhumer.
Portions Copyright (C) 2007-2008 Google.

distcc comes with ABSOLUTELY NO WARRANTY.  distcc is free software, and
you may use, modify and redistribute it under the terms of the GNU 
General Public License version 2 or later.

Please report bugs to distcc@lists.samba.org

Usage:
   distccd [OPTIONS]

Options:
    --help                     explain usage and exit
    --version                  show version and exit
    -P, --pid-file FILE        save daemon process id to file
    -N, --nice LEVEL           lower priority, 20=most nice
    --user USER                if run by root, change to this persona
    --jobs, -j LIMIT           maximum tasks at any time
    --job-lifetime SECONDS     maximum lifetime of a compile request
  Networking:
    -p, --port PORT            TCP port to listen on
    --listen ADDRESS           IP address to listen on
    -a, --allow IP[/BITS]      client address access control
    --stats                    enable statistics reporting via HTTP server
    --stats-port PORT          TCP port to listen on for statistics requests
  Debug and trace:
    --log-level=LEVEL          set detail level for log file
      levels: critical, error, warning, notice, info, debug
    --verbose                  set log level to "debug"
    --no-detach                don't detach from parent (for daemontools, etc)
    --log-file=FILE            send messages here instead of syslog
    --log-stderr               send messages to stderr
    --wizard                   for running under gdb
  Mode of operation:
    --inetd                    serve client connected to stdin
    --daemon                   bind and listen on socket

distccd runs either from inetd or as a standalone daemon to compile
files submitted by the distcc client.

distccd should only run on trusted networks.

```

allow isn't a valid command for distcc (client).

**Comment by carlocab** [comment]
Please provide the output of `brew config` and `brew doctor`. We require this information, relevant or not, from all bug reports at Homebrew.

> Please note we will close your issue without comment if you do not fill out the issue checklist below and provide ALL the requested information.

We can reopen this issue when you've provided the required information.

**Comment by z-aki** [comment]
done

**Comment by z-aki** [comment]
> brew gist-logs <formula> link OR brew config AND brew doctor output

Since either was fine, and gist-logs said "no logs" I thought this would fly
